smarts/core/waypoints.py

```diff
@@ -507,6 +514,7 @@ def _process_interp_for_lane_wp(
                     lane_index=shape_wp.wp.lane_index,
                 ),
                 nexts=[],
+                is_shape_wp=0,
```

Comment:
If this is going to be binary I would prefer `True|False`, otherwise if it has more options it should be an enum.

Reply:
Agree.

Reply:
Thanks, sure.